cocotb_test/simulator.py

```diff
@@ -217,6 +217,11 @@ def set_env(self):
         self.env["TOPLEVEL"] = self.toplevel_module
         self.env["MODULE"] = self.module
 
+        # Force color output, if stdout and stderr are terminals
+        output_isatty = sys.stdout.isatty() and sys.stderr.isatty()
+        if not os.getenv("COCOTB_ANSI_OUTPUT") and output_isatty:
+            self.env["COCOTB_ANSI_OUTPUT"] = "1"
+
```

Comment:
Hrm, `want_color_output` only checks `stdout` in [`cocotb/utils.py`](https://github.com/cocotb/cocotb/blob/master/cocotb/utils.py#L618)

This should probably not check `stderr` and only check `stdout`, to match.

Reply:
Default for cocotb  is "0" so I would not force it here. One can always set this in user code. 
I would remove this part.

Reply:
However, I am seeing color with vanilla `cocotb` but not `cocotb-test`. Attached screen shots for comparison. They show color output running `make` in the standard, unmodified `cocotb/examples/simple_dff/` directory with `COCOTB_ANSI_OUTPUT` not set.

I believe this is because default value of `COCOTB_ANSI_OUTPUT` is not set (which is different than `0` ... it's like `None`), which means "auto", and it will use color if `stdout` is a terminal, as determined by [`isatty()`](https://docs.python.org/3/library/io.html?highlight=isatty#io.IOBase.isatty). Here's the code to `want_color_output()` from `cocotb/utils.py:

```
def want_color_output():
    """Return ``True`` if colored output is possible/requested and not running in GUI.

    Colored output can be explicitly requested by setting :envvar:`COCOTB_ANSI_OUTPUT` to  ``1``.
    """
    want_color = sys.stdout.isatty()  # default to color for TTYs
    if os.getenv("NO_COLOR") is not None:
        want_color = False
    if os.getenv("COCOTB_ANSI_OUTPUT", default="0") == "1":
        want_color = True
    if os.getenv("GUI", default="0") == "1":
        want_color = False
    return want_color
```

Compare to the screen shot when using `include $(shell cocotb-test --inc-makefile)` which uses `cocotb-run`.

In my testing, this change does provide color output by default with `cocotb-test`. Perhaps I should adopt `want_color_output()` verbatim, which would honor `NO_COLOR` and `GUI`?

Color using vanilla `cocotb`:

<img width="1662" alt="cocotb-color" src="https://user-images.githubusercontent.com/20428/231003823-ddba3497-5381-41d4-9ed3-c10987d887fd.png">

No color using `cocotb-test`:

<img width="1662" alt="cocotb-test-no-color" src="https://user-images.githubusercontent.com/20428/231003849-8f87f3f5-90f4-4cac-bcfb-d2bc25fc28ff.png">

Reply:
Okay, I've removed the changes that set `COCOTB_ANSI_OUTPUT`. This PR now only contains the timescale change.